fixtures/configs/learning_registry.json

```diff
@@ -13,8 +13,8 @@
   },
 
   "fts": {
-    "full": ["name", "keywords", "description"],
-    "partial": ["name", "keywords"]
+    "full": ["$.name", "$.keywords", "$.description"],
```

Comment:
We need to check if the config is always retrieved from these fixture files, or if they are loaded on the db the first time and then gets from there. If the last is true, then we will need to change the corresponding db entries on the deploys after merging. Otherwise it's all good.

Reply:
[Looks like](https://github.com/CredentialEngine/CredentialRegistry/blob/master/app/models/envelope_community.rb#L18) the config always comes from the file. I've added some tests to ensure the FTS properties are assigned correctly. Fixed a couple of shortcomings found in the process.